agent-nodejs-8/Dockerfile.rhel7

```diff
@@ -31,6 +32,8 @@ RUN yum-config-manager --enable rhel-server-rhscl-7-rpms && \
     ln -s /usr/lib/node_modules/nodemon/bin/nodemon.js /usr/bin/nodemon && \
     yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
     rpm -V $INSTALL_PKGS && \
+    source /usr/local/bin/scl_enable && \ 
+    npm install -g yarn@${YARN_VERSION} && \
```

Comment:
we can't do this.  all content in the rhel image must come from rpms, we can't source rhel image content from the internet.